db/migrate/20221019102735_add_user_ref_to_likes.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
+  end
```

Comment:
- [ ] Kindly change the name of the foreign key column name to `author` instead of `user`.